npms/skale-observer/observer.js

```diff
@@ -451,14 +454,14 @@ async function cache_schains( strChainNameConnectedTo, w3_main_net, w3_s_chain,
                 cc.debug( "Connected " ) + cc.attention( "S-Chains" ) + cc.debug( " cache was updated in this thread: " ) +
                 cc.j( g_arr_schains_cached ) + "\n" );
         }
-        if( opts.fn_chache_changed )
-            opts.fn_chache_changed( g_arr_schains_cached, null ); // null - no error
+        if( opts.fn_cache_changed )
```

Comment:
Where is the declaration of `fn_cache_changed`?

Reply:
`opts.fn_cache_changed ` is future use, not used now